### PR TITLE
Update 01deg jra55 iaf bgc

### PR DESCRIPTION
**Pull request by minghangli-uni**
Update 01deg_jra55_iaf_bgc configuration. Related https://github.com/ACCESS-NRI/access-om2-configs/issues/48

### CODE REVIEW

**Comment by minghangli-uni** [comment]
@aidanheerdegen It fails at QA test on

`AssertionError: Expect collate executable set to mppnccombine-fast`. 

But from the discussion in the #60, it appears that the decision was made not to add "-fast" for all configurations?

**Comment by aidanheerdegen** [comment]
> But from the discussion in the #60, it appears that the decision was made not to add "-fast" for all configurations?

Definitely for all high res (0.25 and 0.1) models. Just the 1 degree it isn't necessary or even faster.

File: config.yaml [hunk 1]
```diff
@@ -9,19 +9,28 @@
...
-input: /g/data/ik11/inputs/access-om2/input_20201102/common_01deg_jra55
+input:
+    - /g/data/vk83/experiments/inputs/access-om2/remapping_weights/JRA55/global.01deg/2020.05.30/rmp_jra55_cice_1st_conserve.nc
+    - /g/data/vk83/experiments/inputs/access-om2/remapping_weights/JRA55/global.01deg/2020.05.30/rmp_jra55_cice_2nd_conserve.nc
+    - /g/data/vk83/experiments/inputs/access-om2/remapping_weights/JRA55/global.01deg/2020.05.30/rmp_jra55_cice_patch.nc
 submodels:
     - name: atmosphere
       model: yatm
-      exe: /g/data/ik11/inputs/access-om2/bin/yatm_d750b4b.exe
+      exe: /g/data/vk83/apps/spack/0.20/release/linux-rocky8-x86_64/intel-19.0.5.281/libaccessom2-git.2023.10.26=2023.10.26-ieiy3e7hidn4dzaqly3ly2yu45mecgq4/bin/yatm.exe
```
Comment (aidanheerdegen) [inline config.yaml:31]:
```suggestion
      exe: /g/data/vk83/apps/spack/0.20/release/linux-rocky8-x86_64/intel-19.0.5.281/libaccessom2-git.2023.10.26=2023.10.26-ltfg7jcn6t4cefotvj3kjnyu5nru26xo/bin/yatm.exebin/yatm.exe
```

File: config.yaml [hunk 2]
```diff
@@ -37,34 +46,53 @@ submodels:
...
+        - /g/data/vk83/experiments/inputs/access-om2/ice/initial_conditions_biogeochemistry/global.01deg/2022.02.24/o2i.nc
+        - /g/data/vk83/experiments/inputs/access-om2/ice/initial_conditions_biogeochemistry/global.01deg/2022.02.24/i2o.nc
+        - /g/data/vk83/experiments/inputs/access-om2/ice/initial_conditions/global.01deg/2020.05.30/u_star.nc
+        - /g/data/vk83/experiments/inputs/access-om2/ice/initial_conditions/global.01deg/2020.05.30/monthly_sstsss.nc
       ncpus: 722
 
 # Collation
 collate:
   restart: true
-  mpi: true
```
Comment (aidanheerdegen) [inline config.yaml:57]: This should be `mpi: true` with `mppnccombine-fast`. Sorry for the confusion.

**Review by aidanheerdegen — CHANGES_REQUESTED** [review]
Sorry, just a couple more things to fix up.

**Comment by aidanheerdegen** [comment]
The test is failing because the `manifests/exe.yaml` has not been updated with the changed `yatm.exe`. You will need to comment out the 
```yaml
manifest:
  reproduce:
    exe: true
```
part of the `config.yaml` (or set to `false`) and do `payu setup` to update the manifests. Make sure `payu` doesn't add an `experiment_uuid` in this step. If it does this, and makes commit then `git revert` it and remove the `experiment_uuid`.

**Review by aidanheerdegen — APPROVED** [review]
Looks great. Thanks @minghangli-uni

**Pull request merged**